src/offline/OfflineStatus.ts

```diff
@@ -12,13 +14,20 @@ export const getOfflineStatusfromLocalStorage = (): boolean => {
 };
 
 /**
- * Update offline statue on local storage and service worker
+ * Update offline statue on local storage and service worker.
+ * Offline plantid and projectid will also be stored in local storage.
  */
 export const updateOfflineStatus = (
     isOffline: boolean,
-    userPin: string
+    userPin: string,
+    currentProjectId?: number
 ): void => {
-    localStorage.setItem('offline', String(isOffline));
+    localStorage.setItem(OFFLINE_STATUS, String(isOffline));
+
+    //todo: skal vi gjøre dette på en annen måte?
+    if (currentProjectId) {
+        localStorage.setItem(OFFLINE_PROJECT_ID, String(currentProjectId));
```

Comment:
We already store current project and plant in local storage. Could we not use this, since the user is barred from switching project and plant while offline?